chia/consensus/block_creation.py

```diff
@@ -440,8 +429,8 @@ def create_unfinished_block(
         foliage,
         foliage_transaction_block,
         transactions_info,
-        block_generator.program if block_generator else None,
-        [],  # generator_refs
+        new_block_gen.program if new_block_gen else None,
+        new_block_gen.block_refs if new_block_gen else [],
```

Comment:
Was this being missing previously a bug?

Reply:
no. we don't farm block with block references. We used to put the empty list here, but with this patch I made the empty list a member of `NewBlockGenerator`, just as a way to gather the information needed to create the block in there. If we want to use block references at some point, I'm pretty sure it would have to be coordinated by the mempool, and returned via this same path.